experimental/dds/tree2/src/feature-libraries/sequence-field/format.ts

```diff
@@ -309,7 +309,7 @@ export interface MovePlaceholder extends HasRevisionTag, HasMoveId {
 	type: "Placeholder";
 }
 
-export interface TransientEffect extends HasRevisionTag {
+export interface TransientEffect {
```

Comment:
Don't forget to update the `TransientEffect` variable as well.

Reply:
Thanks!